### PR TITLE
Add a short how-to for debugging

### PR DESCRIPTION
**Pull request by vmonkey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,22 @@ We would be happy:
...
+## Debugging
+1. Enable debug logging for the Jablotron intergation via the [logger](https://www.home-assistant.io/integrations/logger/) integration by adding the following lines to the `configuration.yaml` file.
+```
+logger:
+  default: info
+  logs: 
+    custom_components.jablotron100: debug
+```
+
+2. Enable debug logging in the Jablotron integration. Go to the Integration page of your Home Assistant and click on the `Configure` button belonging to the Jablotron integration. Input your security code to the pop-up window and, without changing any of the remaining options, press the `Send` button. Afterwards, a second pop-up window opens: leave that without any change and click on the `Send` button again. In the third pop-up window, enable the debugging by checking the `Enable debugging` checkbox and submit the configuration by pressing the `Send`. Final, fourth pop-up window allows one to specify specific debugging options, such as `Log all incoming packets`. Finish the configuration by pressing the `Send` button.
```
Comment (kukulich) [inline README.md:99]: @vmonkey One small change. There's no need for "Input your security code" - the first two popups can be just skipped with "Send"

**Review by kukulich — APPROVED** [review]

**Comment by kukulich** [comment]
@vmonkey Nice, I planned the same, so I'm really happy somebody write it in English 👍

**Review by kukulich — CHANGES_REQUESTED** [review]
One small change please.

**Review by kukulich — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (vmonkey) [reply README.md]: I did not know that, it should be corrected right now.

**Pull request merged**